### PR TITLE
Cv

### PR DESCRIPTION
**Pull request by christianbrodbeck**
Added a method to evaluate different CV criteria which I use to refit with different mu. In the longer run, do you think there should be the option to haver different criteria, or will we be able to find *the* best one? If you think there will only be one method I could make the function private and remove the stars for the other methods.

### CODE REVIEW

**Comment by christianbrodbeck** [comment]
@proloyd I caught an error introduced in https://github.com/proloyd/DstRF/commit/f320fe0e3dc57f724d5c5d385dbacf1112cf20c9. I modified the tests to run faster and always do cross-validation, so that it will hopefully be easier to catch something like this in the future :). I would recommend setting up Travis.

File: dstrf/_model.py [hunk 1]
```diff
@@ -844,7 +846,7 @@ def eval_obj(self, data):
         -------
             float
         """
-        v = 0
+        residual = 0
```
Comment (christianbrodbeck) [inline dstrf/_model.py]: Maybe there could be better variables names (would be good also for Return part of the docs) I just used what the callers used

**Comment by proloyd** [comment]
Thanks!

File: dstrf/_model.py (hunk 1 above)
Comment (proloyd) [reply dstrf/_model.py]: I was just being lazy. :)

**Pull request merged**